CHANGELOG.rst

```diff
@@ -2,6 +2,52 @@
 SDK Changelog
 =============
 
+3.3.0 (2025-09-30)
+==================
+
+Feature release.
+
+.. note:: this release adds the ability to opt-in to the new experimental form renderer.
+   the feature can be enabled on a per-form basis through the admin interface.
+
+New features
+------------
+
+* [#5191] Added a 'use user location' control to map component.
+* [#5191] Changed the Leaflet-Draw "delete" button behavior and removed the "Delete" menu.
+* [#5575] implemented wms layers in the map component.
+* [#5133] updated the open-formulieren formio-renderer and replaced
+  several components:
+  - select field for appointments.
+  - numberfield.
+* Switched from local storage to session storage for form submissions.
+* [#825] Added children component.
+
+Bugfixes
+--------
+
+* [#871] Added missing translations.
+* [#869] Fixed children component's styling for mobile devices.
+* [#5378] updated map drawing behaviour when only map interaction
+  is allowed.
+* fixed invalid dates in test stories.
```

Comment:
```suggestion
* Fixed invalid dates in test stories.
```
didn't affect users of the SDK, so this should be moved to "Project maintenance"